models/transfers/celo/erc20/transfers_celo_erc20.sql

```diff
@@ -0,0 +1,108 @@
+{{ 
+    config(
+        tags = ['dunesql'],
+        alias = alias('erc20'),
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'tx_hash', 'index', 'wallet_address', 'amount_raw'],
+        post_hook='{{ expose_spells(\'["celo"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["soispoke", "dot2dotseurat", "tschubotz", "tomfutago"]\') }}'
+    )
+}}
+
+with
+    sent_transfers as (
+        select
+            to as wallet_address,
+            contract_address as token_address,
+            evt_block_time as block_time,
+            date_trunc('month', evt_block_time) as block_month,
+            cast(value as double) as amount_raw,
+            evt_index as index,
+            evt_tx_hash as tx_hash
+        from
+            {{ source('erc20_celo', 'evt_transfer') }}
+        where 1=1
+            {% if is_incremental() %} -- this filter will only be applied on an incremental run
+            and evt_block_time >= date_trunc('day', now() - interval '7' day)
+            {% endif %}
+    ),
+    
+    received_transfers as (
+        select
+            "from" as wallet_address,
+            contract_address as token_address,
+            evt_block_time as block_time,
+            date_trunc('month', evt_block_time) as block_month,
+            (-1) * cast(value as double) as amount_raw,
+            evt_index as index,
+            evt_tx_hash as tx_hash
```

Comment:
```suggestion
            evt_index,
            evt_tx_hash as tx_hash,
            'receive' as type
```

Reply:
@Hosuke now that all the tests passed after casting block_month as date do you think the above is still needed?

Reply:
no probs, will add it in